Observatory-Control-System/EAS/DIMM/ops.rst

```diff
@@ -194,5 +210,25 @@ and within the OpenTPL:
 
 Confirm in the monitors that the DIMM operations have ceased. 
 
+Sometimes it may be necessary to manually issue commands to the DIMM dome to close. This can be done by issuing commands to *tt-dome*
```

Comment:
I think this part to close the dome manually should be also inside a box, for instance, using the `admonition` directive. See comment above

Reply:
Fixed in latest version.